dot/node_integration_test.go

```diff
@@ -117,7 +117,7 @@ func TestNewNode(t *testing.T) {
 		return stateSrvc, nil
 	})
 
-	m.EXPECT().createRuntimeStorage(gomock.AssignableToTypeOf(&state.Service{})).Return(&runtime.
+	m.EXPECT().createRuntimeStorage(gomock.AssignableToTypeOf(&state.BaseState{}), gomock.Any()).Return(&runtime.
```

Comment:
Can we not mock the specific type here?

Reply:
Lazy me, done 👍 
Required a bit of test refactoring, but worth it 💯